docs/tutorial/vald-multicluster-on-k8s.md

```diff
@@ -0,0 +1,290 @@
+  ```bash
+  kubectl get vmt -A -o wide
+  NAMESPACE   NAME                                 HOST                                            PORT   STATUS      LAST TRANSITION TIME   AGE
+  vald-03     mirror-target-01                     vald-mirror-gateway.vald-01.svc.cluster.local   8081   Connected   2023-05-22T02:07:51Z   19m
+  vald-03     mirror-target-02                     vald-mirror-gateway.vald-02.svc.cluster.local   8081   Connected   2023-05-22T02:07:51Z   19m
+  vald-02     mirror-target-3296010438411762394    vald-mirror-gateway.vald-01.svc.cluster.local   8081   Connected   2023-05-22T02:07:53Z   19m
+  vald-02     mirror-target-12697587923462644654   vald-mirror-gateway.vald-03.svc.cluster.local   8081   Connected   2023-05-22T02:07:53Z   19m
+  vald-01     mirror-target-13698925675968803691   vald-mirror-gateway.vald-02.svc.cluster.local   8081   Connected   2023-05-22T02:07:53Z   19m
+  vald-01     mirror-target-17825710563723389324   vald-mirror-gateway.vald-03.svc.cluster.local   8081   Connected   2023-05-22T02:07:53Z   19m
+  ```
+
+## Run Example Code
+
+In this chapter, you will execute insert, search, get, and delete vectors to your Vald clusters using the example code.
+
+The [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) is used as a dataset for indexing and search query.
+
+The example code is implemented in Go and uses [vald-client-go](https://github.com/vdaas/vald-client-go), one of the official Vald client libraries, for requesting to Vald cluster.
+
+Vald provides multiple language client libraries such as Go, Java, Node.js, Python, etc.
+
+If you are interested, please refer to [SDKs](https://vald.vdaas.org/docs/user-guides/sdks).
+
+1. Port Forward(option)
+
+   If you do not use Kubernetes Ingress, port-forward is required to make requests from your local environment.
+
+   ```bash
+   kubectl port-forward svc/vald-mirror-gateway 8080:8081 -n vald-01 & \
+     kubectl port-forward svc/vald-mirror-gateway 8081:8081 -n vald-02 & \
+     kubectl port-forward svc/vald-mirror-gateway 8082:8081 -n vald-03
+   ```
+
+2. Download dataset
+
+   Move to the working directory.
+
+   ```bash
+   cd example/client/mirror
+   ```
+
+   Download [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist), which is used as a dataset for indexing and search query.
+
+   ```bash
+   wget https://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+   ```
+
+3. Run Example
+
+   We use [example/client/mirror/main.go](https://github.com/vdaas/vald/blob/feature/mirror-gateway-example/example/client/mirror/main.go) to run the example.
+
+   This example will insert and index 400 vectors into the Vald cluster from the Fashion-MNIST dataset via [gRPC](https://grpc.io/). And then, after waiting for indexing, it will request to search the nearest vector 10 times to all Vald clusters. You will get the 10 nearest neighbor vectors for each search query. And it will request to get vectors using inserted vector id.
```

Comment:
```suggestion
   This example will insert and index 400 vectors into the Vald cluster from the Fashion-MNIST dataset via [gRPC](https://grpc.io/).
   And then, after waiting for indexing, it will request to search the nearest vector 10 times to all Vald clusters. You will get the 10 nearest neighbor vectors for each search query.
   And it will request to get vectors using inserted vector ID.
```